packages/devtools_app/lib/src/analytics/constants.dart

```diff
@@ -108,3 +108,4 @@ const String documentationLink = 'documentationLink';
 // This should track the time from `initState` for a screen to the time when
 // the page data has loaded and is ready to interact with.
 const String pageReady = 'pageReady';
+const String screenReady = 'screenReady';
```

Comment:
can you use the existing constant 'pageReady' for consistency? We can't rename 'pageReady' because there are already analytics being tracked for this constant

Reply:
sure, will need to update dwds later though

Reply:
done!